db/migrate/20240522124545_add_sca_fields_to_proceeding_types.rb

```diff
@@ -0,0 +1,6 @@
+class AddScaFieldsToProceedingTypes < ActiveRecord::Migration[7.1]
+  def change
+    add_column :proceeding_types, :sca_core, :boolean
+    add_column :proceeding_types, :sca_related, :boolean
```

Comment:
Should columns values be `nullable`?

Reply:
No, everything should be true or false and the seed data will ensure that is done